Added initial proxy log repository folder

kb1lqc opened the pull request:
As we aim to get people the ability to test Faraday Software out without the need to purchasing any hardware, we also want to enable open source contributions from anyone whether they own hardware or not. This is one of many log files that will help us round out the ability to run Faraday without any hardware.

The log is described in the text file and will enabled the following programs to be run without hardware:
- Proxy
- Telemetry
- APRS
- SimpleUI

## APRS
While I'm probably asking for people to run this and use KB1LQC-1/2 and essentially make it unusable for me on APRS... if that's the case I'll change out the log file but I will also likely need to avoid using these station IDs in the future on APRS unless we figure out how to indicate otherwise to APRS-IS.

The stations are static and located here on aprs.fi with KB1LQC-1 in Grand Park and KB1LQC-2 in the lower right corner with a green star symbol.

![image](https://cloud.githubusercontent.com/assets/331540/25267293/ff3f997c-2628-11e7-9d69-88a551f192e9.png)

Commit Message:
New folder proxy/examplelogs contains a single log file created for
testing non-hardware use of Faraday software in regards to telemetry. A
new files called logs.txt was created to help document these files and
should be updated as new logs are added. This helps address Issue #168.

kb1lqc committed:
Added initial proxy log repository folder

New folder proxy/examplelogs contains a single log file created for
testing non-hardware use of Faraday software in regards to telemetry. A
new files called logs.txt was created to help document these files and
should be updated as new logs are added. This helps address Issue #168.

kb1lqc committed:
Updated Proxy readme.me for log and test modes

Basic configuration and instructions provided for Proxy logging and test
modes.

reillyeon commented on proxy/examplelogs/logs.txt:
```diff
@@ -0,0 +1,6 @@
+This text document helps define each example log file. These files help users demo as well as test and develop with FaradayRF software without the need for hardware.
+---
+
+Log: 4-20-2017_1.db
```
I recommend standardizing on YYYY-MM-DD-##.db as it sorts better alphabetically.

kb1lqc replied:
Ohh good call.

reillyeon committed:
Fix typo in logs.txt

whiel -> while

kb1lqc committed:
Renamed initial log file for sorting

@reillyeon suggested a better naming scheme for easier sorting of log
files. Makes sense.

kb1lqc committed:
Updated readme.md for Proxy with new log filename

Updated to include the latest initial example log filename.

kb1lqc commented:
Thanks @reillyeon! I just updated the readme.md which I realized had the old filename in it.
